src/App.js

```diff
@@ -0,0 +1,16 @@
+import './App.css'
+import Header from './Header'
+import MainContent from './MainContent'
+import Footer from './Footer'
+
+function App () {
+  return (
+    <div id="page">
+      <Header />
+      <MainContent />
+      <Footer />
+    </div>
+  )
+}
+
+export default App
```

Comment:
Старайся использовать стрелочные функции. И export можно делать сразу
```suggestion
export default () => (
    <div id="page">
      <Header />
      <MainContent />
      <Footer />
    </div>
  )
```

Reply:
поменяла

Reply:
здесь разделила export и объявление функции, так как eslint выдавал ошибку "missing display name"